Header header
bool running
int16 modeID
string modeString
int16 gameStateID
string gameStateString
int16 behStateID
string behStateString
